features/src/main/feature/camel-features.xml

```diff
@@ -1498,6 +1498,16 @@
         <bundle dependency='true'>mvn:io.netty/netty-codec-http/4.1.107.Final</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-olingo4/${project.version}</bundle>
     </feature>
+    <feature name='camel-opensearch' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='[2.16,2.17)'>jackson</feature>
+        <feature version="[4,5)">http-client</feature>
+        <bundle dependency='true'>mvn:jakarta.json/jakarta.json-api/${jakarta-json-api-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:org.apache.httpcomponents/httpasyncclient/${httpasyncclient-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:org.opensearch.client/opensearch-java/${opensearch-java-client-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:org.opensearch.client/opensearch-rest-client-sniffer/${opensearch-rest-client-version}</bundle>
```

Comment:
```suggestion
        <bundle dependency='true'>wrap:mvn:org.opensearch.client/opensearch-rest-client/${opensearch-rest-client-version}</bundle>
        <bundle dependency='true'>wrap:mvn:org.opensearch.client/opensearch-rest-client-sniffer/${opensearch-rest-client-version}</bundle>
```
Warning rest-client was missing, you have been fooled by the split package issue